services/wallet/api.go

```diff
@@ -524,6 +525,37 @@ func (api *API) GetAddressDetails(ctx context.Context, chainID uint64, address s
 	return result, nil
 }
 
+func (api *API) SignMessage(ctx context.Context, message types.HexBytes, address common.Address, password string) (string, error) {
+	log.Debug("[WalletAPI::SignMessage] message: ", message, " address: ", address, " password: ", password)
+	return api.s.transactionManager.SignMessage(message, address, password)
+}
+
+func (api *API) BuildTransaction(ctx context.Context, chainID uint64, sendTxArgsJSON string) (response *transfer.TxResponse, err error) {
+	log.Debug("[WalletAPI::BuildTransaction] chainID: ", chainID, " sendTxArgsJSON: ", sendTxArgsJSON)
```

Comment:
Log API requires the use of a key-value approach to parameters. Otherwise, the entire log line will be an error. Same in the other log calls.
```suggestion
	log.Debug("[WalletAPI::BuildTransaction]", "chainID: ", chainID, " sendTxArgsJSON: ", sendTxArgsJSON)
```

Reply:
Updated, thanks.